CMakeLists.txt

```diff
@@ -173,4 +173,4 @@ install(FILES
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${CMAKE_PROJECT_NAME})
 
 install(EXPORT ${CMAKE_PROJECT_NAME}Targets
-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${CMAKE_PROJECT_NAME})

```

Comment:
`CMAKE_INSTALL_LIBDIR` should work fine actually. It's a relative path, see:
* https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

Reply:
Sounds like a CMake bug, can you try to make an update?

Reply:
No problem, updated cmake to 3.11.1, the same thing, if I use `CMAKE_INSTALL_LIBDIR` in `install(EXPORT)`, tinyxml2Targets.cmake has this in one line:
```cmake
# The installation prefix configured by this project.
set(_IMPORT_PREFIX "/home/<my_path_to_tinyxml>/tinyxml2/_install/gcc-7-cxx17-pic")

# Create imported target tinyxml2
add_library(tinyxml2 SHARED IMPORTED)

set_target_properties(tinyxml2 PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
)
```

Even tried downgrading cmake to 3.0.0. This one actually prints an error (when using `CMAKE_INSTALL_LIBDIR`):
```
CMake Error: install(EXPORT "tinyxml2Targets") given absolute DESTINATION "<cut-out-path>" but the export references an installation of target "tinyxml2" which has relative DESTINATION "lib"
```

Reply:
Oh my...didn't notice that :-) :+1:
That's why `_INCLUDEDIR` in `target_include_directories()` was relative but this one wasn't...

So, I've removed the whole block, hope that's fine...it's a relatively small patch so shouldn't be a maintenance problem?